.github/workflows/lint-autoformat.yml

```diff
@@ -2,9 +2,9 @@ name: Apply lint suggestions
 
 on:
 
-  pull_request:
-    types: [opened, synchronize, reopened]
-    branches: [main]
+  push:
+    tags:
+      - ciflow/autoformat/*
```

Comment:
Ah wait, you also need to add this into https://github.com/pytorch/pytorch/blob/main/.github/pytorch-probot.yml#L3, very easy to miss this step :)